Keycloak/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.2'
+
+services:
+
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: "keycloak"
+      POSTGRES_PASSWORD: ${ppassword}
+      POSTGRES_DB: "keycloak"
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
+    networks:
+      - keycloak
+
+  keycloak:
+    image: jboss/keycloak:4.8.3.Final
+    restart: always
+    depends_on:
+      - postgres
+    ports:
+      - 8443:8443
+    environment:
+      KEYCLOAK_USER: "leAdmin"
+      KEYCLOAK_PASSWORD: ${kcpassword}
+      DB_VENDOR: "postgres"
+      DB_PASSWORD: ${ppassword}
```

Comment:
Keycloak images does not support secret for DB password

Reply:
Noooooooo~